Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook

```diff
@@ -119,12 +64,175 @@
       },
       "name": "tabsGlobal"
     },
-	{
+    {
       "type": 12,
       "content": {
         "version": "NotebookGroup/1.0",
         "groupType": "editable",
         "items": [
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "AMWMetricsUsageDetails\n| summarize c=count(),max(TimeGenerated)\n| project label=iff(c>0,\"Insights as of:\",\"No insights found\"),max_TimeGenerated\n",
+              "size": 4,
+              "timeContext": {
+                "durationMs": 604800000
+              },
+              "queryType": 0,
+              "resourceType": "microsoft.monitor/accounts",
+              "crossComponentResources": [
+                "{Amw}"
+              ],
+              "visualization": "tiles",
+              "gridSettings": {
+                "formatters": [
+                  {
+                    "columnMatch": "Azure Monitor Workspace",
+                    "formatter": 5
+                  },
+                  {
+                    "columnMatch": "Event Count",
+                    "formatter": 4,
+                    "formatOptions": {
+                      "palette": "blue"
+                    }
+                  }
+                ]
+              },
+              "tileSettings": {
+                "titleContent": {
+                  "formatter": 5
+                },
+                "leftContent": {
+                  "columnMatch": "label",
+                  "formatter": 1
+                },
+                "rightContent": {
+                  "columnMatch": "max_TimeGenerated",
+                  "formatter": 6,
+                  "dateFormat": {
+                    "formatName": "longDatePattern"
+                  }
+                },
+                "showBorder": false,
+                "size": "full"
+              }
+            },
+            "name": "freshnessDate",
+            "styleSettings": {
+              "margin": "0 0 -90px 0"
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "AMWMetricsUsageDetails\n| project TimeGenerated,MetricNamespace, MetricName, DimensionsList, DailyTimeseriesCount, IngestedSamplesCount, IncomingEventsCount,DaysSinceMetricQueried,NumberOfQueries\n| summarize argmax(TimeGenerated, *) by MetricNamespace, MetricName, DimensionsList\n| project Namespace = MetricNamespace, Metric=MetricName, Dimensions=DimensionsList,[\"Daily Time Series Count\"]=max_TimeGenerated_DailyTimeseriesCount,[\"Ingested Samples Count\"]=max_TimeGenerated_IngestedSamplesCount,[\"Incoming Events Count\"]=max_TimeGenerated_IncomingEventsCount,[\"Days Since Metrics Queried\"]=max_TimeGenerated_DaysSinceMetricQueried,[\"Number of queries\"]=max_TimeGenerated_NumberOfQueries\n",

```

Comment:
FYI: for these, the suggestion is also to put these into the column labels tab in grid settings to make sure they also get localized